### PR TITLE
Added AB enroll/exit requests in queue

### PR DESCRIPTION
**Pull request by ijunaid**

### CODE REVIEW

**Review by turtledreams — APPROVED** [review]
looks good to me 👍 
we will lint it later for sure
also was curious of the naming of "kCountlyRCKeyKeys"

File: CountlyConnectionManager.m [hunk 1]
```diff
@@ -169,14 +174,22 @@ - (void)proceedOnQueue
         return;
     }
 
-    [CountlyCommon.sharedInstance startBackgroundTask];
-
     NSString* queryString = firstItemInQueue;
+    NSString* endPoint = kCountlyEndpointI;
+    NSString *overrideEndPoint = [self getOverrideTagEndPoint:firstItemInQueue];
```
Comment (ArtursKadikis) [inline CountlyConnectionManager.m:179]: use "queryString"

Comment (ArtursKadikis) [reply CountlyConnectionManager.m:179]: lets refactor this into a function that returns a string array. first item the new query string, second item the extracted endpoint override.

File: CountlyConnectionManager.m [hunk 2]
```diff
@@ -234,6 +254,21 @@ - (void)proceedOnQueue
     [self logRequest:request];
 }
 
+- (NSString *)getOverrideTagEndPoint:(NSString *)queryString
```
Comment (ArtursKadikis) [inline CountlyConnectionManager.m:257]: lets make it static

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]

**Pull request merged**